Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:alpine
+MAINTAINER birgir.stefansson@schibsted.com
+COPY . /ais
+WORKDIR /ais
+RUN pip install -r requirements.txt
+ENTRYPOINT ["python"]
```

Comment:
Since we create entry-points in `setup.py`, the command to run is just `ais`, no need to call python explicitly either.

Reply:
Done